src/Blocks/EntitiesTrait.php

```diff
@@ -203,7 +203,7 @@ protected function mutateBulk(Request $request): Response
     }
 
     /**
-     * Deltes bulk by applying it to transaction/rollback procedure
+     * Deletes bulk by applying it to transaction/rollback procedure
```

Comment:
This is fairly awkward grammatically, but fixing it would be outside the scope of this PR.